src/common/reducers/snap-builds.js

```diff
@@ -30,7 +30,7 @@ export function snapBuilds(state = {}, action) {
         [payload.id]: {
           ...state[payload.id],
           isFetching: false,
-          snapLink: payload.snapLink
+          snap: payload.snap
```

Comment:
Thinking about store structure I'm not sure if `snapBuilds` is the right place to store snap info.

While we use this info on build related pages (build listing and build details) it feels like maybe we should have a separate store for snap data.

We already have snaps (enabled repositories) store, but currently it's an array (so doesn't provide direct access to particular snaps in it) and also snap info fetched for build page may not be from enabled snap of current user (build pages are public). So it feels like completely new store would be needed for that.

But refactoring and restructuring the store feels like a big task that we need to discuss and prioritise separately I guess.